Grammar/src/main/java/com/grammar/TxtFileLog.java

```diff
@@ -23,19 +23,20 @@ public class TxtFileLog {
   private BufferedWriter outFile;
 
   /**
-  * Opens an input file and starts an output file.
+  * This is the constructor with parameters to TxtFileLog object
+  * that opens an input file and starts an output file.
   * 
   * @param fileIn input file name to be opened.
-  * @param fileOut input file name to be started.
+  * @param fileOut  input file name to be started.
   */
   public TxtFileLog(String fileIn, String fileOut){
       openInFile(fileIn);
       startOutFile(fileOut);
   }
   
   /**
-  * Initializes the output file, preparing it for future writes.
-  * @param fileOut path to the output file.
+  * This method is used to initialize the output file, preparing it for future writes.
```

Comment:
I had fixed these doc comments, why are you reverting the changes here? Maybe an accident with `git`?